src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.io.*;
+import java.text.DateFormatSymbols;
+import java.text.SimpleDateFormat;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+//import javafx.application.Application;
+//import javafx.scene.Scene;
+//import javafx.scene.control.Label;
+//import javafx.stage.Stage;
+
```

Comment:
Might be helpful to package all the files so that it will not be confusing for the programmer when the number of files get too big.